### ISSUE TITLE
Turn off verbosity of evaluation set metrics

### ISSUE THREAD

**Issue by singjc** [davechallis/rust-xgboost#24]
Hi,

I am wonder if it's possible to turn off the print statements of the evaluation metrics? These only seem to get printed when a train and test set is provided as evaluation sets?

i.e.

```
[0]     test-auc:0.898594       train-auc:0.917665
[1]     test-auc:0.900673       train-auc:0.926697
[2]     test-auc:0.904338       train-auc:0.934706
[3]     test-auc:0.90517        train-auc:0.938172
[4]     test-auc:0.906147       train-auc:0.942411
```

I have tried using the `verbose(false)` method in the `BoosterParametersBuilder`, and tried setting a `custom_evaluation_fn` in `TrainingParametersBuilder`, but I still get the evaluation metrics printed to the console. 

Thanks, 

Justin

**Comment by singjc** [comment]
Wait, never mind. I think I misunderstood the purpose of the `evaluation_sets` method, which is to evaluate and print the metric.
